configs/votenet/README.md

```diff
@@ -38,3 +38,13 @@ Then you can use the converted checkpoints following [getting_started.md](../../
 ## Indeterminism
 
 Since test data preparation randomly downsamples the points, and the test script uses fixed random seeds while the random seeds of validation in training are not fixed, the test results may be slightly different from the results reported above.
+
+## IoU loss
+
+Adding IoU loss (simply =1-IoU) boosts VoteNet's performance.
+
+|  Backbone   | Lr schd | Mem (GB) | Inf time (fps) | AP@0.25 |AP@0.5| Download |
+| :---------: | :-----: | :------: | :------------: | :----: |:----: | :------: |
+|    [PointNet++](./votenet_8x8_scannet-3d-18class_withiouloss.py)     |  3x    |4.1||63.81|44.21|/|
```

Comment:
votenet_8x8_scannet-3d-18class_withiouloss.py -> votenet_iouloss_8x8_scannet-3d-18class.py
The config name should follow the convention of config systems, see [doc here](https://mmdetection3d.readthedocs.io/en/latest/config.html).